tests/test_client.py

```diff
@@ -204,6 +204,16 @@ async def check_read_write_methods(
         for field_name, c_type in read_struct._fields_:
             assert getattr(read_struct, field_name) == getattr(write_struct, field_name)
 
+        write_str = "terminated data for readuntil"
```

Comment:
Please tweak the data because it is _not_ terminated (and it is for read_str, not readuntil).

Also, if you are willing, adding at least one unicode char would be appreciated -- I should have done that for all the text data in this method.